com.w1hkj.flrig.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>com.w1hkj.flrig</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>flrig</name>
+ <summary>Amateur Radio Rig Control</summary>
+ <developer_name>ARRL, the national association for Amateur Radio</developer_name>
+ <url type="homepage">http://www.arrl.org</url>
+ <description>
+  <p>
+  rig control program, cooperates with fldigi
```

Comment:
Better description from Fedora:
`Flrig is a transceiver control program designed to be used either stand alone or as an adjunct to fldigi.`